lamindb/_record.py

```diff
@@ -891,4 +893,5 @@ def delete(self) -> None:
     }
 
 for name in METHOD_NAMES:
+    attach_func_to_class_method(name, BasicRecord, globals())
     attach_func_to_class_method(name, Record, globals())
```

Comment:
Isn't it enough to add them only to `BasicRecord`? Shouldn't they be inherited then by `Record` without adding them explcitily? Or it doesn't work like that?

Reply:
No, it's surprisingly not enough! See our other discussion on this. I tried it after you raised it yesterday.

Reply:
Ok, thx, good to know.